src/components/PostTypeChoiceModal/PostTypeChoiceModal.tsx

```diff
@@ -119,6 +129,17 @@ const PostTypeChoiceModal = () => {
             <SvgXml xml={pollIcon(theme.colors.base)} width="28" height="28" />
             <Text style={styles.postText}>Poll</Text>
           </TouchableOpacity>
+          <TouchableOpacity
+            onPress={() => onChooseType('livestream')}
+            style={styles.modalRow}
+          >
+            <SvgXml
+              xml={livestreamIcon(theme.colors.base)}
+              width="28"
+              height="28"
+            />
+            <Text style={styles.postText}>Livestream</Text>
```

Comment:
Please use i18n for all text (if it already setup, please check with @NaingAmity )